### DISCUSSION TITLE
Multiselect filterable has set the Clear all selected items button to aria-hidden which means screen readers can't see it

### DISCUSSION THREAD

**Discussion by scottdickerson** [carbon-design-system/carbon#8488] [Help] [answered]
In previous versions, the MultiSelect.Filterable component set the aria role and label correctly for the 'Clear all selected items' button.  But in this release, the aria-hidden tag has been set to true, which means that screen readers will skip it.  This seems like a functionality we need to expose to screen readers.  I don't see an alternate way to clear the selected items all at once.

Notice the aria-hidden true here in the accessibility pane
![image](https://user-images.githubusercontent.com/6663002/115754184-2bf7a980-a359-11eb-9581-2ef75c21827a.png)

## Environment

> Operating system

> Browser
Chrome

> Automated testing tool and ruleset
@testing-library/react (the issue was actually found by my test automation)

> Assistive technology used to verify

See above in Chrome and the testing library

> What version of the Carbon Design System are you using?
carbon-components-react 7.31.0
carbon-components@10.31.0

> What did you expect to happen?
In previous versions aria-hidden wasn't set for this button

**Comment by dakahn** [comment, marked as answer]
This was an intentional decision for three reasons:
1. a nested interactable inside an input results in a poor screen reader experience
2. the clear button itself is a redundant visual affordance (potentially unnecessary). keyboard/screen reader users looking to clear a selection can either use our built in <kbd>esc</kbd> functionality or the browser default <kbd>cmd</kbd>/<kbd>ctrl</kbd>+<kbd>a</kbd>, <kbd>backspace</kbd>
3. with a nested interactable in the input itself the next tab stop is _inside_ the component itself and not _the next_ form element/link/whatever. This is potentially confusing UX.

I'll convert this to a discussion for reference/to continue the conversation 👍🏽

**Reply by sunnyanand17** [reply]
So for the violation reported on the above by IBM Equal Access Accessibility Checker, what should we do? Should we treat these violations as false positives? Should we accept them as Not Applicable? 
Discussion in Carbon Channel: https://ibm-systems-z.slack.com/archives/C046Y0YUD/p1634776657291700
Discussion in AVT Channel: https://ibm-systems-z.slack.com/archives/C036P1CTN/p1634778022371100
